.gitlab-ci.yml

```diff
@@ -0,0 +1,17 @@
+stages:
+  - build
+
+cross-build:
+  image: registry.fedoraproject.org/fedora:latest
+  stage: build
+  script:
+    - dnf install -y make git-core golang podman
```

Comment:
`golang` is needed for the variables that `Makefile` sets:

```
GOOS ?= $(shell go env GOOS)
GOARCH ?= $(shell go env GOARCH)
HOST_BUILD_DIR=$(BUILD_DIR)/$(GOOS)-$(GOARCH)
GOPATH ?= $(shell go env GOPATH
```

`git-core` is needed for:
```
COMMIT_SHA=$(shell git rev-parse --short HEAD)
```
and
```
SOURCES := $(shell git ls-files  *.go ":^vendor"
```